gdcdictionary/schemas/copy_number_auxiliary_file.yaml

```diff
@@ -0,0 +1,86 @@
+$schema: "http://json-schema.org/draft-04/schema#"
+
+id: "copy_number_auxiliary_file"
+title: Copy Number Auxiliary File
+type: object
+namespace: https://gdc.cancer.gov
+category: data_file
+project: '*'
+program: '*'
+description: >-
+  Data file related to the copy number pipeline that contains any outputs not strictly defined in other nodes
+additionalProperties: false
+submittable: false
+downloadable: true
```

Comment:
jira says downloadable:false (but guessing ticket is wrong?)

Reply:
Changed JIRA.